src/accesspattern/arrays.jl

```diff
@@ -0,0 +1,24 @@
+typealias NativeArray Union{Array,SubArray}
```

Comment:
I'm a little hesitant here... what about the many other array types, like PermutedDimsArray?